README.md

```diff
@@ -124,6 +124,15 @@ if (!globalThis.fetch) {
 }
 ```
 
+Stuck with commonjs and can't use ESM imports?
+This is a ESM-only module. Using require to load an ES module is not supported because ES modules have asynchronous execution. Instead, use `import()` to lazy load an ES module from a CommonJS module only when needed.
+It can improve startup performance
+
+```js
+// mod.cjs
+const fetch = (...args) => import('node-fetch').then(mod => mod.default(...args));
```

Comment:
```suggestion
const fetch = (...arguments_) => import('node-fetch').then(({default: fetch}) => fetch(...arguments_));
```

Reply:
I did not like the name `arguments_` it's to similar to arguments that is a reserved keyword.
think `args` is fine and common enough. 

if anything else then it should be
`const fetch = () => import('node-fetch').then(({default: fetch}) => fetch(...arguments))`

Reply:
I don't think

```
const fetch = () => import('node-fetch').then(({default: fetch}) => fetch(...arguments))
```

Would work since `arguments` will refer to the inner function, and not the outer one.

-----

Personally, I prefer the version that was first added, that is with `...args` and `mod.default(...args)`.

Reply:
> I did not like the name `arguments_` it's to similar to arguments that is a reserved keyword.

I like having full variable names since it makes the code more readable. Even if it seems easy enough, [you can never be so sure](https://www.youtube.com/watch?v=QPZ0pIK_wsc&t=171s).

Reply:
> Would work since arguments will refer to the inner function, and not the outer one.

Ah, right... stupid mistake :P

Reply:
In which case,

```suggestion
const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```